addons/mrp/wizard/change_production_qty.py

```diff
@@ -86,7 +86,7 @@ def change_prod_qty(self):
                     quantity = quantity if (quantity > 0) else 0
                 if float_is_zero(quantity, precision_digits=precision):
                     wo.check_ids.unlink()
-                else:
+                elif wo.qty_producing:
```

Comment:
It risks to be a litle strange also with this usecase:
- WO enterprise
- Start the first WO, set 1 in qty producing, do some check
- Update quantity with the wizard.

Shouldn't we do a method to check if the MO quantity should be update with an overwrite in OE.
Also the wo.check_ids.unlink() seems strange...